builds/e2e/connectivity.yaml

```diff
@@ -19,38 +19,43 @@ jobs:
           testrun.network.runProfile: "Online"
           testrun.network.frequencies: "00:00:00 00:00:00 0"
           testrun.duration: "01:00:00"
-        Offline(5/5/5):
+        Offline:
           testrun.config: 2
+          testrun.network.runProfile: "Offline"
```

Comment:
1. Remove this case OFFLINE
2. Introduce the dup + backlog items to deploy w/ separate connectivity deployment file (use foreach log).